### PR TITLE
TRITON-2149 Update vmapi to use the workflow job status change event

### PR DESCRIPTION
**Pull request by twhiteman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.9.5",
+  "version": "9.10.0",
```
Comment (kusor) [inline package.json:4]: Wondering if this change is big enough for `10.y.z` here ...

File: docs/index.md [hunk 2]
```diff
@@ -2258,6 +2258,70 @@ Returns a job with the specified UUID.
       ]
     }
 
+## WaitJob (GET /jobs/:uuid/wait)
+
+Waits until the given job with the specified UUID is finished (i.e. when the
+job execution state is one of "succeeded", "failed" or "canceled").
```
Comment (kusor) [inline docs/index.md:2264]: Does this mean that the request will not return a reply until the job is finished?

File: package.json (hunk 1 above)
Comment (twhiteman) [reply package.json:4]: There is no change to the vmapi functionality (and polling is still used as the fallback, same as before) - so I'm thinking 9.10 is fine.

File: docs/index.md (hunk 2 above)
Comment (twhiteman) [reply docs/index.md:2264]: Correct.

**Pull request merged**